core/src/mast/node/basic_block_node/arbitrary.rs

```diff
@@ -175,7 +175,7 @@ impl Arbitrary for BasicBlockNode {
             })
             .prop_map(|(ops, decorators)| {
                 // BasicBlockNode::new will adjust indices for padding and set be/ae empty.
-                BasicBlockNode::new(ops, decorators)
+                BasicBlockNode::new_owned_with_decorators(ops, decorators)
```

Comment:
I think the comment here should mention `BasicBlockNode::new_owned_with_decorators` rather than `BasicBlockNode::new`